<launch>

  <node pkg="stage" type="stageros" name="stageros" args="-g $(find r3po)/stage/lab.world" respawn="false" ns="stage1">
    <remap from="/clock" to="/stage1/clock"/>
    <param name="base_watchdog_timeout" value="0.2"/>
  </node>
  <node pkg="r3po" type="robot_pose.py" name="robot_pose" ns="stage1">
  </node>
  
  <node pkg="stage" type="stageros" name="stageros" args="-g $(find r3po)/stage/lab.world" respawn="false" ns="stage2">
    <remap from="/clock" to="/stage2/clock"/>
    <param name="base_watchdog_timeout" value="0.2"/>
  </node>
  <node pkg="r3po" type="robot_pose.py" name="robot_pose" ns="stage2">
  </node>
  
  <node pkg="stage" type="stageros" name="stageros" args="-g $(find r3po)/stage/lab.world" respawn="false" ns="stage3">
    <remap from="/clock" to="/stage3/clock"/>
    <param name="base_watchdog_timeout" value="0.2"/>
  </node>
  <node pkg="r3po" type="robot_pose.py" name="robot_pose" ns="stage3">
  </node>
  
  <node pkg="stage" type="stageros" name="stageros" args="-g $(find r3po)/stage/lab.world" respawn="false" ns="stage4">
    <remap from="/clock" to="/stage4/clock"/>
    <param name="base_watchdog_timeout" value="0.2"/>
  </node>
  <node pkg="r3po" type="robot_pose.py" name="robot_pose" ns="stage4">
  </node>
  
</launch>
